src/content/sessions/2023-09-01.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Rider: CE custom operations completion list"
+preview: ""
+isDraft: true
+date: 2023-09-01T13:00:00.000Z
```

Comment:
Unfortunately, 2023-09-01 is not possible for me. How about 2023-09-08 ? Any time until 16:00.

Reply:
Sure that works for us